src/shared/components/ProfilePage/Stats/DistributionGraph/index.jsx

```diff
@@ -0,0 +1,288 @@
+/* eslint-env browser */
+import d3 from 'd3';
+import React from 'react';
+import { toPairs } from 'lodash';
+import PT from 'prop-types';
+import ChartTooltip from '../ChartTooltip';
+import './index.scss';
+
+const getRanges = distribution => (
+  toPairs(distribution)
+    .map(([name, number]) => ({
+      name,
+      number,
+      start: parseInt(name.match(/\d+/)[0], 10),
+      end: parseInt(name.match(/To(\d+)/)[1], 10),
+    }))
```

Comment:
Not sure about the exact purpose of this code piece, but I feel the same can be done better.